src/FormContainer.tsx

```diff
@@ -16,8 +16,17 @@ const makeWrapper = <T extends {}>(config: IFormConfig<T>) => (WrappedComponent:
             };
         }
 
-        setModel = (model: { [name in keyof T]: any }) => {
-            this.setState({ model });
+        setModel = (model: { [name in keyof T]: any }, setTouched: boolean = false) => {
+            let touched: { [key: string]: boolean } = this.state.touched;
+
+            if (!setTouched) {
+                this.setState({ model });
+            } else {
+                for (var field in model) {
+                    touched[field] = true;
```

Comment:
there's a `setFieldToTouched` method exposed as public API on the form. Will it help to solve the problem?